test/test/boot.js

```diff
@@ -449,6 +449,41 @@ async function checkSupport() {
   }
 }
 
+function checkClearKeySupport() {
+  const clearKeySupport = shakaSupport.drm['org.w3.clearkey'];
+  if (!clearKeySupport) {
+    return false;
+  }
+  return clearKeySupport.encryptionSchemes.includes('cenc');
+}
+
+function checkPlayReadySupport() {
+  return shakaSupport.drm['com.microsoft.playready'] ||
+      shakaSupport.drm['com.microsoft.playready.recommendation'] ||
+      shakaSupport.drm['com.microsoft.playready.recommendation.3000'];
+}
+
+function checkWidevineSupport() {
+  return shakaSupport.drm['com.widevine.alpha'];
+}
+
+function checkFairPlaySupport() {
+  return shakaSupport.drm['com.apple.fps'] && !getClientArg('runningInVM');
+}
+
+function checkWidevinePersistentSupport() {
+  const widevine = checkWidevineSupport();
+  if (!widevine) {
+    return false;
+  }
+  return widevine.persistentState;
+}
+
+function checkTrueDrmSupport() {
+  // We don't include FairPlay because our test assets aren't ready to use it.
+  return checkWidevineSupport() || checkPlayReadySupport();
+}
```

Comment:
add jsdoc comments for better readability

Reply:
Done!